roles/nodenv_install/tasks/main.yaml

```diff
@@ -7,4 +7,4 @@
 - name: node installation messages
   debug:
     msg: "{{nodenv_install.stderr}}"
-  when: nodenv_install.stderr
+  when: nodenv_install.stderr | length > 0
```

Comment:
Related to the [breaking changes on Ansible 12](https://docs.ansible.com/ansible/latest/porting_guides/porting_guide_12.html#broken-conditionals). Need to check if we have other cases in `ofn-install`.